smak/Server/netController.cpp

```diff
@@ -239,3 +257,52 @@ void smak::netController::opInfo(const rapidjson::Document &jsonDom, std::shared
 
     fromUser.get()->sendString("The server you are on is: " + Name + "\nThis server has been running since: "+beginTime+"\nThe current time is: "+curTime+"\nFor more info use OP code LIST");
 }
+
+void smak::netController::opPing(const rapidjson::Document &jsonDom, std::shared_ptr<smak::User> fromUser) {
+    std::string Name = serverState->getName();
+    std::string beginTime = serverState->getStartTime();
+
+    fromUser.get()->sendString("[PONG] Your connection is good, server is: "+Name+"\nThis server has been running since: "+beginTime);
```

Comment:
I'm not sure if this is in the RFC, but it might be nice to say the current server time. In a real world scenario servers are often in different timezones or a few seconds off from their clients.